draft-ietf-rats-corim.md

```diff
@@ -1207,6 +1207,242 @@ The following describes each member of the `concise-bom-tag` map.
+which are used as input to appraisal by the Verifier.
+
+### Cryptographic validation of Evidence
+
+If the authenticity of Evidence is secured by a cryptographic mechanism such as
+a signature, the first step in the Evidence Appraisal is to perform
+cryptographic validation of the Evidence.
+
+The exact cryptographic signature validation mechanics depend on the specific
+Evidence collection protocol.
+
+For example:
+In DICE, a proof of liveness is performed on the final key in the certificate
+chain.
+If this passes then a suitable certification path anchored on a trusted root
+certificate is looked up -- e.g., based on linking information obtained from
+the DeviceID certificate (see Section 9.2.1 of {{DICE-Layering-Architecture}}) --
+in the Appraisal Context.  If found, then usual X.509 certificate validation
+is performed.
+In PSA, the verification public key is looked up in the appraisal context using
+the `euid` claim found in the PSA claims-set (see {{Section 4.2.1 of -psa-token}}).
+If found, COSE Sign1 verification is performed accordingly.
+
+Independent of the specific integrity protection method used, the integrity of
+Evidence MUST be successfully verified.
+
+> A CoRIM profile MUST describe:
+>
+> * How cryptographic verification key material is represented (e.g., using Attestation Keys triples, or CoTS tags)
+> * How key material is associated with the Attesting Environment
+> * How the Attesting Environment is identified in Evidence
+
+### The Accepted Claims Set
+
+At the end of the Evidence collection process evidence has been converted into
+a format suitable for appraisal. This document describes an `accepted-claims-set`
+format and the algorithms used to compare it against CoMID reference values.
+
+    accepted-claims-set = {​
+        &(ce.evidence-triples: 0) => [ + reference-triple-record ]​
+      ? &(ce.identity-triples: 1) => [ + identity-triple-record ]​
+      ? &(ce.dependency-triples: 2) => [ + domain-dependency-triple-record ]
+      ? &(ce.domain-membership-triples: 3) => [ + domain-membership-triple-record ]
+      ? &(ce.coswid-triples: 4) => [ + ev-coswid-triple-record ]
+      * $$ev-triples-map-extension​
+    }
+
+Verifiers are not required to use this as their internal state, but for the
+purposes of this document a sample Verifier is discussed which uses this format.
+
+The Accepted Claims Set will be matched against CoMID reference values, as per
+the appraisal policy of the Verifier.
+This document describes an example evidence structure which can be easily
+matched against these reference values.
+Each set of evidence contains an `environment-map` providing a namespace, and
+a non empty `measurement-values-map`.
+
+Each entry in the `measurement-values-map` is a separate piece of evidence
+describing the environment named by the `environment-map`.
+
+ An Attester can provide multiple `environment-map`s each containing a
+ `measurement-values-map` with one entry;  a single `environment-map` containing
```

Comment:
100 % agree with statement on line 1360 and hence we should support this expression natively in the base CDDL.